src/instructlab/config/init.py

```diff
@@ -161,13 +165,23 @@ def init(
     )
 
 
-def check_if_configs_exist() -> bool:
+def gate_user_from_overwriting_config():
```

Comment:
Please retain the return type hint (now it's None).

Reply:
Not only is the type hint gone but so is the return 😭

Reply:
The return of None is implicit in python, so that's fine, I guess.